src/antsibull_changelog/cli.py

```diff
@@ -8,13 +8,15 @@
 Entrypoint to the antsibull-changelog script.
 """
 
+from __future__ import annotations
+
 import argparse
 import datetime
 import os
 import sys
 import traceback
 
-from typing import Any, Callable, cast, List, Optional, Tuple
+from typing import Any, Callable, cast, List
```

Comment:
`collections.abc.Callable` should be used instead.

``` console
$ rg 'from typing import.*Callable'
stubs/tomli.pyi:from typing import Callable, Any
src/antsibull_changelog/utils.py:from typing import Any, Callable
src/antsibull_changelog/cli.py:from typing import Any, Callable, cast, List
src/antsibull_changelog/changes.py:from typing import Any, Callable, cast
tests/functional/fixtures.py:from typing import Any, Callable
```

Reply:
`typing.List` is used along with `cast()` in this file. Those can use `list` as well. That is available at runtime on Python 3.9 and above.

Reply:
0f3295ce0448eb53c8b795f74597202a3fb6df65.